vignettes/Visualisations.Rmd

```diff
@@ -0,0 +1,507 @@
+---
+title: "Visualisations"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 3
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Visualisations}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This Vignette provides a preliminary overview of visualisations for forecasts and scores. Section [Visualising forecasts] gives an overview of how to visualise forecasts. Section [Visualising scores] provides an overview of how to visualise scores. Section [Old visualisations] provides the code for visualisations that were previously part of `scoringutils` but have now been removed.
```

Comment:
I think for the moment we should be making it clear this vignette is showing how to recreate visualisations that have been depreciated vs trying to make it seem like a visualisation gallery? Perhaps we could make this two issues 1. THis one that is the depreciated visualisation vignette which we will eventually depreciate and then another that showcases some visualisations we like (and in that none not break package rules like using `:::`?

Reply:
Yeah good point, agree. I updated the vignette to reflect that.